assets/js/atomic/blocks/product-list/image/index.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { registerBlockType } from '@wordpress/blocks';
+import Gridicon from 'gridicons';
+import { Fragment } from 'react';
```

Comment:
Here we could use `@wordpress/element` instead of `react`, right?
```suggestion
import { Fragment } from '@wordpress/element';
```